packages/tools/grub/config/grub.cfg

```diff
@@ -0,0 +1,21 @@
+set timeout="5"
+set default="3"
+menuentry "LibreELEC Boot Menu"{
+	set
+}
+menuentry ' '{
+	set
+}
+menuentry "1. Installer"{
+	search --set -f /KERNEL
+	linux /KERNEL boot=LABEL=LIBREELEC installer quiet tty vga=current
+}
+
+menuentry "2. Live"{
+	search --set -f /KERNEL
+	linux /KERNEL boot=LABEL=LIBREELEC live quiet tty vga=current
+}
+menuentry "3. Portable"{
+	search --set -f /KERNEL
+	linux /KERNEL boot=LABEL=LIBREELEC disk=UUID=132e735c-fc99-4639-8138-c8db4f7698b5 portable quiet

```

Comment:
I don't think the UUID should be specified?

Reply:
Agreed, this should be `$UUID_STORAGE`. The boot argument could also use `UUID=$UUID_SYSTEM` (obviously the variables need to be expanded).

Also, just a nit: in the syslinux installer we refer to the 3rd mode as `run` - just to avoid introducing another term that complicates support, could we call the 3rd menuentry `3. Portable / Run`

Reply:
I think I understand what you mean. I can make adjustment to mkimage to create grub.cfg using variables but my issue is which branch I should push the commit to automatically link back to this PR?

Reply:
You created changes from your local libreelec-8.2 branch. In future always make a topic branch, e.g. after `git checkout master` do `git checkout -b new_branch` so you avoid making changes in the main release branches. On this occasion you can get away with it because most of your PR is new files being added and the only existing file being changed is identical in both branches so your changes apply cleanly. Anyway.. make the required changes in `scripts/mkimage` then do `push -f origin libreelec-8.2` to update this PR.